src/ADAL.PCL.CoreCLR/PlatformInformation.cs

```diff
@@ -61,5 +61,20 @@ public override string GetDeviceModel()
             // Since ADAL .NET may be used on servers, for security reasons, we do not emit device type.
             return null;
         }
+
+        public override string GetApplicationName()
```

Comment:
Its a abstract method in base class

---
In reply to: [72841188](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r72841188) [](ancestors = 72841188)